st2common/setup.py

```diff
@@ -53,6 +53,8 @@
         "bin/st2-register-content",
         "bin/st2-purge-executions",
         "bin/st2-purge-trigger-instances",
+        "bin/st2-purge-trace",
```

Comment:
For consistency it would be good to use plural form of the name here as well - e.g. purge-traces and purge-rule-enforcements.